decidim/decidim
# Fix error message mismatch in election

#### :tophat: What? Why?
After we merged https://github.com/decidim/decidim/pull/10900, we have noticed there are some pipelines that are failing in the election area. On a closer look, we have noticed that the error message is "The setup is being done at least 1 **hours** before the election starts."
This PR fixes the error message to have a correct pluralization, but also the pipeline.

#### :pushpin: Related Issues
- Related to #10900

#### Testing
* See the pipeline is green 
* visit the admin area, election setup and check the message is indeed: `The setup is being done at least 1 hour before the election starts.`

:hearts: Thank you!
